### PR TITLE
add instructions to replicate per-passage baseline

### LINKED ISSUE #40: MS MARCO doc retrieval per-passage baseline
We have MS MARCO doc, per passage expansion - which gives good results. However, for a fair comparison, we need per passage retrieval, but _without_ expansion.

### PR DESCRIPTION
**Pull request by richard3983**
fixes #40

### CODE REVIEW

File: convert_msmarco_passages_doc_to_anserini.py [hunk 1]
```diff
@@ -28,8 +28,9 @@ def create_segments(doc_text, max_length, stride):
     description='Concatenate MS MARCO original docs with predicted queries')
 parser.add_argument('--original_docs_path', required=True, help='MS MARCO .tsv corpus file.')
 parser.add_argument('--doc_ids_path', required=True, help='File mapping segments to doc ids.')
-parser.add_argument('--predictions_path', required=True, help='File containing predicted queries.')
 parser.add_argument('--output_docs_path', required=True, help='Output file in the anserini jsonl format.')
+parser.add_argument('--predictions_path', help='File containing predicted queries.')
+parser.add_argument('--no_expansion', default=False, type=bool, help='expand with predicted queries or not')
```
Comment (rodrigonogueira4) [inline convert_msmarco_passages_doc_to_anserini.py:33]: How about remove this argument and do not add expansions in case predictions_path==None

**Comment by lintool** [comment]
btw, I'm trying out these instructions to replicate. will report back results.

File: convert_msmarco_passages_doc_to_anserini.py (hunk 1 above)
Comment (richard3983) [reply convert_msmarco_passages_doc_to_anserini.py]: yes, I changed that

File: convert_msmarco_passages_doc_to_anserini.py [hunk 2]
```diff
@@ -43,11 +43,14 @@ def create_segments(doc_text, max_length, stride):
 nlp = spacy.blank("en")
 nlp.add_pipe(nlp.create_pipe("sentencizer"))
 
-print('Appending segments...')
+print('Spliting documents...')
 doc_id_ref = None
-for doc_id, predicted_queries_partial in tqdm(zip(open(args.doc_ids_path),
-                                                      open(args.predictions_path))):
-    doc_id = doc_id.strip()
+if args.predictions_path == None:
```
Comment (rodrigonogueira4) [inline convert_msmarco_passages_doc_to_anserini.py:48]: just a minor comment, it is better to to:
```
if args.predictions_path:
    ...
else:
    ...
```
This way you cover both `args.predictions_path == None` and `args.predictions_path == ''`

**Review by rodrigonogueira4 — APPROVED** [review]
LGTM! Just a minor comment. Feel free to merge if you think it is relevant.

**Comment by lintool** [comment]
I was able to replicate the results. Merge when ready!

**Review by rodrigonogueira4 — APPROVED** [review]

**Pull request merged**